.travis.yml

```diff
@@ -5,53 +5,12 @@ install:
   - pip install --upgrade pip tox
 
 script:
-  - (cd core && tox -e py27)
-  - (cd bigtable && tox -e py27)
-  - (cd storage && tox -e py27)
-  - (cd datastore && tox -e py27)
-  - (cd bigquery && tox -e py27)
-  - (cd pubsub && tox -e py27)
-  - (cd logging && tox -e py27)
-  - (cd dns && tox -e py27)
-  - (cd language && tox -e py27)
-  - (cd error_reporting && tox -e py27)
-  - (cd resource_manager && tox -e py27)
-  - (cd monitoring && tox -e py27)
-  - (cd vision && tox -e py27)
-  - (cd translate && tox -e py27)
-  - (cd speech && tox -e py27)
-  - (cd core && tox -e py34)
-  - (cd bigtable && tox -e py34)
-  - (cd storage && tox -e py34)
-  - (cd datastore && tox -e py34)
-  - (cd bigquery && tox -e py34)
-  - (cd pubsub && tox -e py34)
-  - (cd logging && tox -e py34)
-  - (cd dns && tox -e py34)
-  - (cd language && tox -e py34)
-  - (cd error_reporting && tox -e py34)
-  - (cd resource_manager && tox -e py34)
-  - (cd monitoring && tox -e py34)
-  - (cd vision && tox -e py34)
-  - (cd translate && tox -e py34)
-  - (cd speech && tox -e py34)
+  - python2.7 scripts/run_unit_tests.py
+  - python3.4 scripts/run_unit_tests.py
+  - python3.5 scripts/run_unit_tests.py
```

Comment:
I tried to use `python3.5` out of the box and it wasn't present. I don't want to introduce a matrix (with more than one job) in this PR, which is what is required in the link you included. (I was also misled by that link.)

We do have #2350 open for the 3.5 / Travis discussion